Dockerfile

```diff
@@ -1,9 +1,12 @@
-FROM alpine:3.14
+ARG ALPINE_VERSION=3.14
+FROM alpine:${ALPINE_VERSION}
 LABEL Maintainer="Tim de Pater <code@trafex.nl>"
 LABEL Description="Lightweight container with Nginx 1.20 & PHP 8.0 based on Alpine Linux."
+# Setup document root
+WORKDIR /var/www/html
 
 # Install packages and remove default server definition
-RUN apk --no-cache add \
+RUN apk add --no-cache --update \
```

Comment:
Why are you adding the `--update` flag?

Reply:
Update repository indexes.

Reply:
I think this is not needed since `--no-cache` already ignores the local cache if there was any.

Reply:
You're right, i already removed it